templates/miq-template.yaml

```diff
@@ -88,7 +88,7 @@ objects:
             persistentVolumeClaim:
               claimName: ${NAME}
         containers:
-        - image: fbladilo/miq-app:latest
+        - image: fbladilo/miq-app:${APPLICATION_IMG_TAG}
```

Comment:
@fbladilo Not really related to the change itself... but where does this value get used?  It seems the value used in the actual Deployments screen is what comes from the ImageStream 'name' and I can't seem to find where this is actually used.